operations/iuf/workflows/management_rollout.md

```diff
@@ -211,10 +211,37 @@ Refer to that table and any corresponding product documents before continuing to
         --no-config-change --config-name "${CFS_CONFIG_NAME}" --xnames $MASTER_STORAGE_XNAMES --clear-state
         ```
 
-        The expected output is:
+        The sample output for a management node is:
```

Comment:
Suggesting to change this line since it is configuring multiple management nodes.
```suggestion
       Sample output for configuring multiple management nodes is:
```

Reply:
made the change